lib/resources/r4/soarian_allergy_intolerance.yaml

```diff
@@ -0,0 +1,83 @@
+name: AllergyIntolerance
+field_name_base_url: http://hl7.org/fhir/r4/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
```

Comment:
Two changes:
- Delete the second "the" so the sentence reads "The type of FHIR resource."
- Add a registered trademark symbol after "FHIR".